plugins/Transitions/API.php

```diff
@@ -156,13 +156,19 @@ public function getTransitionsForAction($actionName, $actionType, $idSite, $peri
 
     /**
      * Derive the action ID from the request action name and type.
+     *
+     * @param string $actionName
+     * @param string $actionType
+     *
+     * @return array|int|string
      */
-    private function deriveIdAction($actionName, $actionType)
+    private function deriveIdAction(string $actionName, string $actionType)
```

Comment:
you need to add that to the public api methods as well, as only those are automatically checked and handled...

Reply:
Thanks, clearly wasn't thinking there  :man_facepalming: :slightly_smiling_face: